src/gql/generated/types.ts

```diff
@@ -164,7 +165,7 @@ export type Mutation = {
   setTaskPriority: Task;
   restartTask: Task;
   saveSubscription: Scalars["Boolean"];
-  removePatchFromCommitQueue?: Maybe<Scalars["String"]>;
+  RemoveItemFromCommitQueue?: Maybe<Scalars["String"]>;
```

Comment:
once you change this to camelcase in Evergreen, you'll have to `npm run codegen` again to change it here.